dask_expr/io/tests/test_parquet.py

```diff
@@ -561,6 +561,15 @@ async def test_parquet_distriuted(c, s, a, b, tmpdir, filesystem):
     assert_eq(await c.gather(c.compute(df.optimize())), pdf)
 
 
+def test_pickle_size(tmpdir, filesystem):
+    pdf = pd.DataFrame({"x": [1, 4, 3, 2, 0, 5]})
+    [_make_file(tmpdir, df=pdf, filename=f"{x}.parquet") for x in range(10)]
+    df = read_parquet(tmpdir, filesystem=filesystem)
+    from distributed.protocol import dumps
+
+    assert len(b"".join(dumps(df.optimize().dask))) <= 8300
```

Comment:
this was at 19500 before so this is a reduction in size of about 60%

Reply:
> this was at 19500 before so this is a reduction in size of about 60%

Did you mean ~60x?

Reply:
I don't follow? 19.5k to 8.3k is roughly 60% less

Reply:
Never mind, I typed too many zeros.